micro-ui/web/micro-ui-internals/packages/modules/core/src/components/RoleBasedEmployeeHome.js

```
@@ -127,100 +160,65 @@ export const RoleBasedEmployeeHome = ({ modules, additionalComponent }) => {
-              variation="teritiary"
-              label={configureData?.label}
-              icon={configureData?.icon}
-              type="button"
-              size={"medium"}
-              onClick={() => history?.push(configureData?.link)}
-              style={{ padding: "0px" }}
-            />,
-            <Button
+      links: Digit.Utils.getMultiRootTenant() ? getFilteredLinks(moduleData) : moduleData.links,
+      centreChildren: [
+        <div>{t(Digit.Utils.locale.getTransformedLocale(`MODULE_CARD_DESC_${current}`))}</div>,
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)